src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlService.java

```diff
@@ -239,6 +252,46 @@ public CaseEventTrigger setReadOnlyOnCaseViewFieldsIfNoAccess(final CaseEventTri
         return caseEventTrigger;
     }
 
+    public CaseEventTrigger updateCollectionDisplayContextParameterByAccess(final CaseEventTrigger caseEventTrigger,
+                                                                            final Set<String> userRoles) {
+        caseEventTrigger.getCaseFields()
```

Comment:
consider one of these alternative options:
1) introduce a method on CaseEventTrigger called getCollectionFields
2) caseEventTrigger.getCaseFields().filter(caseViewField.isCollectionFieldType())....

Reply:
Going for option 2, because introducing getCollectionFields is tricky. We have two methods, one with List<CaseViewField> and one with List<CaseFiled>. It is easier to do the filter manually.